helm_deploy/offender-case-notes/values.yaml

```diff
@@ -43,10 +43,10 @@ generic-service:
       DATABASE_PASSWORD: "database_password"
       DATABASE_NAME: "database_name"
       DATABASE_ENDPOINT: "rds_instance_endpoint"
-    offender-events-topic:
-      HMPPS_SQS_TOPICS_OFFENDEREVENTS_ACCESS_KEY_ID: "access_key_id"
-      HMPPS_SQS_TOPICS_OFFENDEREVENTS_SECRET_ACCESS_KEY: "secret_access_key"
-      HMPPS_SQS_TOPICS_OFFENDEREVENTS_ARN: "topic_arn"
+    domain-events-topic:
+      HMPPS_SQS_TOPICS_DOMAIN_EVENTS_ACCESS_KEY_ID: "access_key_id"
```

Comment:
I think you might need to drop the underscore in `DOMAIN_EVENTS`. Spring converts these env vars into properties (in this case the matching property is `domain-events` which also neeeds to change). But we've found that sometimes Spring gets tripped up by separators so to be safe we don't use them on queue and topic names.

Reply:
Thanks, good call - I've updated those.

I believe this is to do with k8s not handling environment variables with hyphens.  See https://github.com/kubernetes/kubernetes/issues/16863